packages/scripts/src/verifyPagesInSource.ts

```diff
@@ -0,0 +1,132 @@
+/**
+  Verify that there is at least 1 page for each source in the Pages collection. 
+  This acts as a sanity check that no sources failed to copy/ingest
+*/
+import { assertEnvVars, makeMongoDbPageStore } from "mongodb-rag-core";
+import "dotenv/config";
+
+const { MONGODB_DATABASE_NAME, MONGODB_CONNECTION_URI } = assertEnvVars({
+  MONGODB_DATABASE_NAME: "",
+  MONGODB_CONNECTION_URI: "",
+});
+
+const excludeSources: Set<string> = new Set([
+  "snooty-guides",
+  "snooty-atlas-cli",
+  "snooty-docs-k8s-operator",
+  "snooty-c",
+  "snooty-mongodb-vscode",
+  "snooty-intellij",
+  "snooty-entity-framework",
+  "snooty-node",
+  "snooty-cpp-driver",
+  "snooty-csharp",
+  "snooty-mongocli",
+  "snooty-pymongo",
+  "snooty-rust",
+  "snooty-mongodb-shell",
+  "snooty-django",
+  "snooty-scala",
+  "snooty-cloud-manager",
+  "snooty-atlas-architecture",
+  "snooty-charts",
+  "snooty-ruby-driver",
+  "snooty-cloud-docs",
+  "snooty-landing",
+  "snooty-golang",
+  "snooty-bi-connector",
+  "snooty-kafka-connector",
+  "snooty-java",
+  "snooty-kotlin-sync",
+  "snooty-drivers",
+  "snooty-mongoid",
+  "snooty-cluster-sync",
+  "snooty-docs-relational-migrator",
+  "snooty-atlas-operator",
+  "snooty-spark-connector",
+  "snooty-compass",
+  "snooty-ops-manager",
+  "snooty-kotlin",
+  "snooty-php-library",
+  "snooty-cloudgov",
+  "snooty-visual-studio-extension",
+  "snooty-laravel",
+  "snooty-database-tools",
+  "snooty-pymongo-arrow",
+  "snooty-java-rs",
+  "mongodb-dot-com",
+  "web-company",
+  "web-customer-case-studies",
+  "web-solutions-library",
+  "web-university",
+  "web-services",
+]);
+
+async function main() {
```

Comment:
To check my understanding - this script finds sources that at one point existed in our collection but have since been deleted without any replacements from the same source.

I think this approach is valid just want to make sure that's what we're doing.